node/tests/ui_gateway_test.rs

```diff
@@ -49,27 +34,21 @@ fn ui_requests_something_and_gets_corresponding_response() {
                     home_dir.into_os_string().to_str().unwrap(),
                 ),
         ),
-        false,
+        true,
         true,
         false,
         true,
     );
     node.wait_for_log("UIGateway bound", Some(5000));
-    let financials_request = UiFinancialsRequest {};
+    let descriptor_request = UiCheckPasswordRequest {
```

Comment:
This should either really be a descriptor request, or be named for the check password request.